fail2ban/server/datetemplate.py

```diff
@@ -64,7 +64,7 @@ def name(self, name):
 	def getRegex(self):
 		return self._regex
 
-	def setRegex(self, regex, wordBegin=True):
+	def setRegex(self, regex, wordBegin=True, wordEnd=True):
```

Comment:
There is even a docstring with description of wordbegin... Would be nice to note the wordend there I guess